packages/dialtone-vue2/vite.config.js

```diff
@@ -99,13 +100,50 @@ export default defineConfig({
     },
   },
   test: {
+    name: 'dialtone-vue2',
     globals: true,
     environment: 'jsdom',
     setupFiles: './tests/setupTests.js',
     exclude: ['common/custom-emoji.test.js'],
     include: ['./{common,components,directives,recipes}/**/*.test.js'],
     coverage: {
-      reporter: ['text', 'html'],
+      provider: 'v8',
+      reporter: ['text', 'html', 'json'],
+      reportsDirectory: './coverage',
```

Comment:
How will you get this in the KPI reports if it's not written to the repo or the output of the package?